### DISCUSSION TITLE
Evaluating Accuracy of the Model

### DISCUSSION THREAD

**Discussion by sandeepmohanadasan** [AlexeyAB/darknet#7861] [General]
I have a set of Images and Yolo Annotation files(in txt format) for the validation.

How to use -map argument on validating the Images to get the mAP score? Will it be possible to derive the confusion matrix from this?

@AlexeyAB @stephanecharette  Could you be able to throw some light on this?

**Comment by DeemAQ** [comment]
have you figured it out?
